web/packages/shared/utils/createMfaOptions.ts

```diff
@@ -16,31 +16,23 @@
  * along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
 
-import { Auth2faType, PreferredMfaType } from 'shared/services/types';
-
-export default function createMfaOptions(opts: Options) {
-  const { auth2faType, required = false } = opts;
-  const mfaOptions: MfaOption[] = [];
-
-  if (auth2faType === 'off' || !auth2faType) {
-    return mfaOptions;
-  }
-
-  const mfaEnabled = auth2faType === 'on' || auth2faType === 'optional';
+import { getMfaRegisterOptions } from 'teleport/services/mfa';
 
-  if (auth2faType === 'webauthn' || mfaEnabled) {
-    mfaOptions.push({ value: 'webauthn', label: 'Passkey or Security Key' });
-  }
-
-  if (auth2faType === 'otp' || mfaEnabled) {
-    mfaOptions.push({ value: 'otp', label: 'Authenticator App' });
-  }
-
-  if (!required && auth2faType === 'optional') {
-    mfaOptions.push({ value: 'optional', label: 'None' });
-  }
+import { Auth2faType, PreferredMfaType } from 'shared/services/types';
 
-  return mfaOptions;
+// Deprecated: use getMfaRegisterOptions or getMfaChallengeOptions instead.
+// TODO(Joerger): Delete once no longer used.
+export default function createMfaOptions(opts: Options): MfaOption[] {
```

Comment:
Now that this no longer handles `required`, should this just accept `auth2faType`?

Reply:
Yeah, but I can't change the function signature until the RecoveryFlow in /e starts using the new MfaOptions methods. I'm going to remove the method all existing calls to `createMfaOptions` in a couple of follow up PRs to avoid merge conflicts and /e shenanigans.